### PR TITLE
Add triangle scene

### PR DESCRIPTION
**Pull request by cristeahub**
Very early stage of triangle scene.

### CODE REVIEW

**Comment by cristeahub** [comment]
Not a triangle yet, but trust me it will be lol

File: src/triangleNode.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+(function(global) {
+  class triangleNode extends NIN.ShaderNode {
+    constructor(id, options) {
+      console.log("options for triangle", options);
+      options.shader = SHADERS.triangle;
+
+      this.quad.material.uniforms.
```
Comment (stianjensen) [inline src/triangleNode.js]: This will be fun 😄

Comment (cristeahub) [inline src/triangleNode.js]: Fixed it now. My computer crashed so this slipped through lol

Comment (cristeahub) [inline src/triangleNode.js]: Loads of "lols" nowadays. Probably because Lejla is watching Paradise Hotel in the background.

**Pull request merged**